modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path

```diff
@@ -10,6 +10,11 @@
 </thead>
 
 <tbody>
+	<tr>
+		<td>ALLERT_CONFIRMATION_BUTTON</td>
+		<td>//button[contains(@class,'btn-warning') and text()='${key_buttonType}']</td>
+		<td></td>
+	</tr>
```

Comment:
@JhossephG did we use this anywhere? If so, maybe we can use an existing and more generic locator instead of creating a new one?

Reply:
I removed this commit, I actually was already using another path that does the same, I just forgot to remove, thanks Patty!